api/src/main/java/org/openmrs/Person.java

```diff
@@ -416,7 +420,7 @@ public void setNames(Set<PersonName> names) {
 	 */
 	public List<PersonAttribute> getActiveAttributes() {
```

Comment:
So this function here is the same as [`BaseCustomizableData#getActiveAttributes()`](https://github.com/openmrs/openmrs-core/blob/de7bfbb8f84ccf82ea5fd4e934c7a9ba9e483f31/api/src/main/java/org/openmrs/BaseCustomizableData.java#L50), which means we can remove this function altogether from the `Person` class since it's implemented by the `BaseCustomizableData` class.